climpred/constants.py

```diff
@@ -1,102 +1,59 @@
-# metrics to be used in compute_hindcast
-DETERMINISTIC_HINDCAST_METRICS = [
-    'pearson_r',
-    'pearson_r_p_value',
-    'rmse',
-    'mse',
-    'mae',
-    'msss_murphy',
-    'conditional_bias',
-    'bias',
-    'std_ratio',
-    'bias_slope',
-    'nmae',
-    'nrmse',
-    'nmse',
-    'ppp',
-    'uacc',
-    'spearman_r',
-    'spearman_r_p_value',
-    'mape',
-    'smape',
-    'mad',
-]
+from .comparisons import __ALL_COMPARISONS__ as all_comparisons
+from .metrics import __ALL_METRICS__ as all_metrics
+
+# to match a metric for (multiple) keywords
+METRIC_ALIASES = dict()
+for m in all_metrics:
+    if m.aliases is not None:
+        for a in m.aliases:
+            METRIC_ALIASES[a] = m.name
+
+
+DETERMINISTIC_METRICS = [m.name for m in all_metrics if not m.probabilistic]
 
+# TODO: remove hindcast_PM_metric thing
```

Comment:
What does this mean?

Reply:
hindcast_metrics and PM_metrics was introduced when we had `less` which didnt work for PM. 

in principle we wouldn't need that differentiation. it doesnt harm to have it like this now. its not really a todo. I will remove that comment.